Quadriga/src/main/webapp/WEB-INF/views/transformation/transformation.jsp

```diff
@@ -0,0 +1,84 @@
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1"
+	pageEncoding="ISO-8859-1"%>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core"%>
+<%@ taglib uri="http://www.springframework.org/tags" prefix="spring"%>
+
+
+<!--  
+	Author Jaydatta Nagarkar, Jaya-Venkata Vutukuri  
+	Used to list the networks
+-->
+
+<script type="text/javascript">
+	$(document).ready(function() {
+		$("ul.pagination1").quickPagination({
+			pageSize : "10"
+		});
+		$("ul.pagination2").quickPagination({
+			pageSize : "10"
+		});
+
+	});
+</script>
+<script type="text/javascript" charset="utf8">
+	$(document).ready(function() {
+		activeTable = $('.dataTable').dataTable({
+			"bJQueryUI" : true,
+			"sPaginationType" : "full_numbers",
+			"bAutoWidth" : false
+		});
+	});
+	$(document).ready(function() {
+		$("input[type=button]").button().click(function(event) {
+			event.preventDefault();
+		});
+	});
+</script>
+
+
+<header>
+	<h2>Transformations</h2>
+	<span class="byline">List of approved Networks</span>
+</header>
+<span class="byline">Networks you have approved.</span>
```

Comment:
The page should show all approved networks, not just the ones that the user has approved.